plugin/session_view.py

```diff
@@ -202,11 +202,12 @@ def diagnostics_tag_scope(self, tag: int) -> Optional[str]:
         return None
 
     def present_diagnostics_async(self) -> None:
-        flags = 0 if userprefs().show_diagnostics_highlights else sublime.DRAW_NO_FILL | sublime.DRAW_NO_OUTLINE
+        flags = userprefs().diagnostics_highlight_style_flags()  # for single lines
+        multiline_flags = 0 if userprefs().show_multiline_diagnostics_highlights else sublime.DRAW_NO_FILL | sublime.DRAW_NO_OUTLINE
```

Comment:
I don't get this `show_multiline_diagnostics_highlights`, do you also want to add the option to not render multiline diagnostics at all?

Reply:
Yeah, I believe the user should have the option to disable all highlights. If they just set `diagnostics_highlight_style` to `""`, it will (currently) only disable the single-line highlights. So this option was added to explicitly toggle multiline highlights.

The reasons I did not want to infer this option from the single setting are the following:
1. Checking that the value `""` is fine, but we would also need to handle the case when all severities are set to `""`. If we choose not to, we would create a mismatch between the meaning of value `""` and an object with all the values set to `""`.
2. Relatedly, it would be better if the user expects `diagnostics_highlight_style` to either deal with all highlights, or only single line (as they are now, maybe we should rename it?). The former gives a wrong expectation that the user can customize multi-line highlight styles. So I chose to create a separate setting.